src/spectra/tests/test_uniform.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+import enoki as ek
+import mitsuba
+
+
+def make_spectrum(value=None, lambda_min=None, lambda_max=None):
+    from mitsuba.core.xml import load_string
```

Comment:
Would be great to use `load_dict` instead of `load_string` in the unit tests. This enables better readibility.

You can have a look at the doc [here](https://mitsuba2.readthedocs.io/en/latest/src/python_interface/parsing_xml.html#creating-objects-using-python-dictionaries)

The plan is to slowly rewrite all the tests so that they use `load_dict`.